new-content/learning-resources/textbook-1-3.md

```diff
@@ -1,7 +1,7 @@
 ---
 order: 100
-title: Qiskit Textbook Chapters 1 - 3
-to: https://qiskit.org/textbook/ch-states/introduction.html
+title: Introduction Course
```

Comment:
This is part of another issue: https://github.com/Qiskit/qiskit.org/issues/1932, could you remove these changes?